src/main/docker/RNApeg.sh

```diff
@@ -78,6 +75,10 @@ fi
 ##################
 
 echo "[*] Running junction_extraction_wrapper.pl"
-junction_extraction_wrapper.pl -no-config -bam $BAMFILE -o $OUTPUT_DIR -fasta $FASTA -refflat $REFFLAT -refgene $REFGENE -now
-
+if [[ -z "$OUTPUT_DIR" ]]; then
+    junction_extraction_wrapper.pl -no-config -bam $BAMFILE -fasta $FASTA -refflat $REFFLAT -refgene $REFGENE -now
+else
+    mkdir -p $OUTPUT_DIR
+    junction_extraction_wrapper.pl -no-config -bam $BAMFILE -o $OUTPUT_DIR -fasta $FASTA -refflat $REFFLAT -refgene $REFGENE -now
```

Comment:
It's a stylistic preference, but I'd prefer to do something like `$output_arg` which is always passed to the command invocation. It is either empty, or contains the directory and argument if the directory was specified.

Reply:
So you want `$output_arg` to contain `-o dir_name` (or nothing) and remove the if statement, right?